tools/haskell-translator/lhs_pars.py

```diff
@@ -17,6 +17,7 @@
 from functools import reduce
 from msgs import error, warning
 
+
```

Comment:
Is this an intentional newline?

Reply:
yes, it's supposed to be its own section for (local) imports, and needs to be 2 lines away from the `class` that follows

Reply:
Although I should fish out the `braces` import that is in the middle of the system imports above. Will do.